### ISSUE TITLE
Could use some file icons

### ISSUE THREAD

**Issue by guiri14** [mattstermiller/koffee#3]

**Comment by mattstermiller** [comment]
Agreed, the bolder font and Type columns isn't enough to easily distinguish the folders from the files. Initially, we can just add generic folder and file icons, but loading the icons from Windows would be nice too.

**Comment by jilleJr** [comment]
Want to give a tip on this guide: https://docs.microsoft.com/en-us/dotnet/desktop/winforms/advanced/how-to-extract-the-icon-associated-with-a-file-in-windows-forms?view=netframeworkdesktop-4.8

> When displaying files in a list control or table control, you may want to display the icon representing the file type next to each file name. You can do this easily by using the [ExtractAssociatedIcon](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.icon.extractassociatedicon?view=netframework-4.5.1) method.

Pretty basic in theory.

**Comment by mattstermiller** [comment]
Thanks. I want to add that the icon loading should be performed in the background to not slow down the app. The folders should also have nice icons, obtained from Windows itself if possible.
